.github/workflows/release.yml

```diff
@@ -0,0 +1,101 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.0
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@5e15885530fb01d81d1f24e8a6f54ebbd0fed7eb
```

Comment:
yes: immutable SHA. 👍🏻 

Perhaps mention intended version?

Also, 2.6.1 is out; should we grab that upgrade right now? 🤷🏻

Reply:
I've added a comment to specify which version it belongs to. I haven't bumped it yet. I would like to test that version bump first although reading the release notes it should be really safe (they seem like chore related changes). Assuming everything still works, i'll bump all the tools to keep goreleaser action in sync across the tools.